business/openshift_oauth.go

```diff
@@ -203,8 +205,15 @@ func (in *OpenshiftOAuthService) Logout(token string) error {
 		return fmt.Errorf("could not connect to Openshift: %v", err)
 	}
 
+	// convert the access token to the corresponding oauthaccesstoken resource name
+	// see: https://github.com/openshift/console/blob/9f352ba49f82ad693a72d0d35709961428b43b93/pkg/server/server.go#L609-L613
```

Comment:
Do we need to note that this changed in OpenShift 4.6 and so if run on earlier versions the new approach will fail like the old approach does on 4.6+?   Should we silently try the old delete if the new one returns an error?

Reply:
we could try to silently delete if this fails. I can do that. Actually, let me just do the same thing I did in the 1.12 PR for all of these. That way is one code base and will work everywhere.

Reply:
this is done. I'll update the backport PRs now so they use the same code.